dvc/remote/local.py

```diff
@@ -257,26 +262,119 @@ def wrapper(from_info, to_info, *args, **kwargs):
 
 class LocalRemote(BaseRemote):
     scheme = Schemes.LOCAL
-    PARAM_CHECKSUM = "md5"
-    PARAM_PATH = "path"
-    TRAVERSE_PREFIX_LEN = 2
     INDEX_CLS = RemoteIndexNoop
     TREE_CLS = LocalRemoteTree
 
-    UNPACKED_DIR_SUFFIX = ".unpacked"
-
+    PARAM_CHECKSUM = "md5"
+    PARAM_PATH = "path"
     DEFAULT_CACHE_TYPES = ["reflink", "copy"]
+    TRAVERSE_PREFIX_LEN = 2
+    UNPACKED_DIR_SUFFIX = ".unpacked"
 
     CACHE_MODE = 0o444
 
-    def __init__(self, repo, config):
-        super().__init__(repo, config)
-        self.cache_dir = config.get("url")
-
     @property
     def state(self):
         return self.repo.state
 
+    def get(self, md5):
```

Comment:
Is this function needed? It looks like it was leftover from some old behavior, and is local only (not found in `BaseRemote`). It's still being tested in `tests/func/test_data_cloud` but I couldn't find anywhere else in DVC that it's actually used.